src/main/java/org/apache/jmeter/protocol/aws/cognito/CognitoProducerAdminCreateUser.java

```diff
@@ -0,0 +1,111 @@
+     */
+    @Override
+    public Arguments getDefaultParameters() {
+        Arguments defaultParameters = new Arguments();
+        defaultParameters.setArguments(Stream.of(AWS_PARAMETERS, COGNITO_PARAMETERS)
+                .flatMap(List::stream)
+                .collect(Collectors.toList()));
+        return defaultParameters;
+    }
+
+    /**
+     * Main method to execute the test on single thread. Create new user in Cognito.
+     * @param context
+     *        Arguments values on Java Sampler.
+     * @return SampleResult, captures data such as whether the test was successful,
+     * the response code and message, any request or response data, and the test start/end times
+     */
+    @Override
+    public SampleResult runTest(JavaSamplerContext context) {
+
+        SampleResult result = newSampleResult();
+        sampleResultStart(result, String.format("Cognito Pool Id: %s " + 
+                        "\nUser Username : %s \nUser Email: %s",
+                context.getParameter(COGNITO_USER_POOL_ID),
+                context.getParameter(COGNITO_USER_USERNAME),
+                context.getParameter(COGNITO_USER_EMAIL)));
+
+        try{
+            log.info("Create user in Cognito");
+            AdminCreateUserResponse createUserRsp = cognitoClient.adminCreateUser(createAdminCreateUserRequest(context));
+            cognitoClient.adminSetUserPassword(createAdminSetUserPasswordRequest(context)); // this will also set user status to confirmed
+            
+            sampleResultSuccess(result, String.format("User Attributes: %s", createUserRsp.user().attributes()));            
+        } catch (CognitoIdentityProviderException exc){
+            log.error("Unable to create user " + context.getParameter(COGNITO_USER_USERNAME), exc);
+            sampleResultFail(result, exc.awsErrorDetails().errorCode(), exc.awsErrorDetails().errorMessage());
+        } catch (JsonProcessingException exc) {
+            log.error("Unable to create user " + context.getParameter(COGNITO_USER_USERNAME), exc);
+            sampleResultFail(result, FAIL_CODE, exc.getMessage());
+        }
+
+        return result;
+    }
+    
+    protected AdminCreateUserRequest createAdminCreateUserRequest(JavaSamplerContext context) 
+            throws JsonProcessingException {
+        AttributeType attributeType = AttributeType.builder()
+                .name("email")
+                .value(context.getParameter(COGNITO_USER_EMAIL))
+                .build();      
+        
+        return AdminCreateUserRequest.builder()
+                .userPoolId(context.getParameter(COGNITO_USER_POOL_ID))
+                .username(context.getParameter(COGNITO_USER_USERNAME))
+                .userAttributes(attributeType)
+                .messageAction("SUPPRESS")
+                .build();
+    }
+    
+    protected AdminSetUserPasswordRequest createAdminSetUserPasswordRequest(JavaSamplerContext context) 
+            throws JsonProcessingException {
```

Comment:
You can remove the throw statement because the code is not deserializing JSON string to Java object.